src/ProductsXmlFeed.php

```diff
@@ -84,18 +84,25 @@ public static function get_xml_footer() {
 	/**
 	 * Returns the Item's XML for the given product.
 	 *
-	 * @param WC_Product $product  The product to print the XML for.
-	 *
-	 * @return string XML string
+	 * @param  WC_Product $product  The product to print the XML for.
+	 * @param  string     $location The location to print the XML for.
+	 * @return string     XML string.
 	 */
-	public static function get_xml_item( $product ) {
+	public static function get_xml_item( $product, $location ) {
```

Comment:
The `$location` variable isn't used. Is this something that is meant for a future change?

Reply:
Yes, it will allow us to control what shipping information is sent in Feed for the particular location and it will allow the creation of logic for currency swap and other types of internationalization.